baseline/OSPS-BR.yaml

```diff
@@ -307,6 +319,11 @@ controls:
           - reference-id: MA-8
           - reference-id: SI-4
           - reference-id: SI-5
+      - reference-id: UKSSCOP
+        entries:
+          - reference-id: Claim 1.1.4
+          - reference-id: Claim 2.2.3
+          - reference-id: Claim 3.1.1
```

Comment:
I don't see the connection here. I read 3.1.1 as being about cryptographic hash validation.

Reply:
logging changes assists consumers in verifying the integrity of the software they are deploying and understanding the full impact of those changes to their systems

Reply:
I agree with Ben -- the claim is that "the _integrity_ of software (including updates)..." (my emphasis).  I don't think a descriptive log of the changes passes that test; I'd expect something involving hashes and signatures.  (e.g. BR-06)